.travis.yml

```diff
@@ -4,17 +4,20 @@ cache: ccache
 os: linux
 matrix:
   include:
+    - name: "First commit message adheres to guidelines at https://goo.gl/p2fr5Q"
+      language: node_js
+      node_js: "node"
+      script:
+        - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
+            bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST};
```

Comment:
if you split this into `install` & `script` the script could fold.
```yaml
      install:
        - RET=0
          if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
            bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST} > out.txt
            RET=$?
          fi
          export RET
      script:
         - cat out.txt || true
           exit $RET
```

Reply:
Redirecting stdout to file will lose coloring (okay not a major issue 😛). But if we do want to allow folding perhaps the thing to do is split the shell script so that the first part works out the first commit (this can be the `install` (`before_script`?) step) and then `script` would directly run `npx core-validate-commit ...`.

Reply:
As in `install` == `fetch from GitHub`...
It will be a nicer UX, but will make the script more complex.
Your call.